db/migrate/20190507152056_create_rents.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :rents do |t|
+      t.belongs_to :user
+      t.belongs_to :book
```

Comment:
If it's null? 🤔

Reply:
cant be, im doing a validation when i create a new Rent